portal/static/js/sessionMonitor.js

```diff
@@ -37,107 +20,113 @@ var SessionMonitorObj = function() {
                     logoutUrl: n + "/logout",
                     timeoutUrl: n + "/logout?timeout=1",
                     ping: function() {
-                      $.ajax({
-                          type: "POST",
-                          contentType: "text/plain",
-                          headers: {
-                              "X-CSRFToken": o
-                          },
-                          cache: !1,
-                          url: l.pingUrl,
-                          crossDomain: !0
-                      });
+                        $.ajax({
+                            type: "POST",
+                            contentType: "text/plain",
+                            headers: {
+                                "X-CSRFToken": o
+                            },
+                            cache: !1,
+                            url: l.pingUrl,
+                            crossDomain: !0
+                        });
                     },
                     logout: function() {
-                      if (typeof(sessionStorage) !== "undefined") {
-                        sessionStorage.clear();
-                      }
-                      window.location.href = l.logoutUrl;
+                        if (typeof(sessionStorage) !== "undefined") {
+                            sessionStorage.clear();
+                        }
+                        window.location.href = l.logoutUrl;
                     },
                     onwarning: function() {
-                      var n = Math.round(l.timeBeforeWarning / 60 / 1e3),
-                          o = $('<div id="jqsm-warning">Your session will expire in ' + n + ' minutes. <button id="jqsm-stay-logged-in">Stay Logged In</button><button id="jqsm-log-out">Log Out</button></div>');
-                      $("body").children("div#jqsm-warning").length || $("body").prepend(o), $("div#jqsm-warning").show(), $("button#stay-logged-in").on("click", function(n) {
-                          n && n.stopPropagation(), l.extendsess(n);
-                      }).on("click", function() {
-                          o.hide();
-                      }), $("button#jqsm-log-out").on("click", l.logout);
+                        var n = Math.round(l.timeBeforeWarning / 60 / 1e3),
+                            o = $("<div id='jqsm-warning'>Your session will expire in " + n + " minutes. <button id='jqsm-stay-logged-in'>Stay Logged In</button><button id='jqsm-log-out'>Log Out</button></div>");
```

Comment:
Should this be translated?

Reply:
@ivan-c ivan, this is default html if custom one is not provided - which we do provide and the custom html is tagged for translation.